docs/podman-create.1.md

```diff
@@ -396,8 +396,7 @@ its root filesystem mounted as read only prohibiting any writes.
 
 **--rm**=*true*|*false*
    Automatically remove the container when it exits. The default is *false*.
-   `--rm` flag can work together with `-d`, and auto-removal will be done on daemon side. Note that it's
-incompatible with any restart policy other than `none`.
+   `--rm` flag can work together with `-d`, and auto-removal will be done on daemon side. 
```

Comment:
Actually, as long as you're in here, why not just remove this entirely or change it to `--rm flag is not compatible with -d` ? Same with run.